python/ray/tune/execution/tune_controller.py

```diff
@@ -154,8 +154,20 @@ def _wrapped(self):
         return TrialRunnerWrapper(
             self,
             trial_executor=_FakeRayTrialExecutor(self),
-            runner_whitelist_attr={"search_alg", "get_trials", "_set_trial_status"},
-            executor_whitelist_attr={"has_resources_for_trial", "pause_trial", "save"},
+            runner_whitelist_attr={
+                "search_alg",
```

Comment:
```suggestion
                "_search_alg",
```

Reply:
These are attributes that the developer of the schedulers can use - they shouldn't access private properties

Reply:
Oh, I didn't see that `search_alg` was a property defined in the base tune controller, makes sense!